designer/client/src/fixedPortal.tsx

```diff
@@ -0,0 +1,14 @@
+import { css } from "@emotion/css";
+import React from "react";
+
+export const FixedPortal = () => (
```

Comment:
Maybe we should rename this component to say explicitly that, it's needed for the glide data editor? I had a problem with understanding what was going on here and why, but finally i just found out that the library requires this snippet... https://docs.grid.glideapps.com/api/dataeditor#html-css-prerequisites